### ISSUE TITLE
Block Click-Through Referrer only working when Default Mode is Allow

### ISSUE THREAD

**Issue by GoogleCodeExporter** [bliu886/scriptno#59]
```
What steps will reproduce the problem?
1. Turn on Block Click-Through Referrer in Options
2. Set Default Mode to Allow in Options
3. Use any search engine and search for b2knet.com
4. Click link for b2knet.com
5. On bottom of page the referring search engine and search is shown

What is the expected output? What do you see instead?
I would expect the Block Click-Through Referrer option to not depend on the 
Default Mode. 

What version of the product are you using? On what operating system?
ScriptNo: 1.0.5.52
Chrome Beta: 18.0.1025.11 beta-m
Windows 7 32bit

Please provide any additional information below.

```

Original issue reported on code.google.com by `dpr...@gmail.com` on 12 Feb 2012 at 1:06

**Comment by GoogleCodeExporter** [comment]
Original comment by `andr...@gmail.com` on 15 Feb 2012 at 5:17
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Please try v1.0.6.0 and let me know if this is solved now :)
```

Original comment by `andr...@gmail.com` on 16 Feb 2012 at 3:33
- Changed state: **FeedbackNeeded**

**Comment by GoogleCodeExporter** [comment]
```
It does seem to be resolved in 1.0.6.0 with a few different tests I have tried.

Thanks for the quick fix!

David
```

Original comment by `dpr...@gmail.com` on 16 Feb 2012 at 5:12

**Comment by GoogleCodeExporter** [comment]
```
Great, thanks :)
```

Original comment by `andr...@gmail.com` on 16 Feb 2012 at 5:22
- Changed state: **Done**
